packages/nimble-components/src/wafer-map/modules/prerendering.ts

```diff
@@ -131,13 +132,27 @@ export class Prerendering {
     }
 
     private calculateOpacity(
-        selectedValue: string,
-        highlightedValues: Readonly<string[]>
+        dieTags?: string[],
+        highlightedTags?: string[]
     ): number {
-        return highlightedValues.length > 0
-            && !highlightedValues.some(dieValue => dieValue === selectedValue)
-            ? this.nonHighlightedOpacity
-            : 1;
+        if (
+            highlightedTags?.length === 1
+            && highlightedTags?.every(tag => tag === '')
```

Comment:
i think you do not have to check for '' tags as that is a valid value in the dieTags list

Reply:
you are right